<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>livre_dialog</class>
 <widget class="QDialog" name="livre_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1155</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1161</width>
     <height>581</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color : #181DA4 ;
border-radius : 20px ;
color : white ;
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/prefix1/10.png</normaloff>:/new/prefix1/10.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Livre Liste</string>
    </attribute>
    <widget class="QLabel" name="label_status_livre_bd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>520</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>451</width>
       <height>491</height>
      </rect>
     </property>
     <property name="title">
      <string>Livre Info</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Code Livre</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_code">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Titre</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_titre"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Nbr_pages</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_nbrp"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Auteur</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_auteur"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Etat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_etat"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_init">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>440</y>
        <width>60</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/46.png</normaloff>:/new/prefix1/46.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>25</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>380</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/59.png</normaloff>:/new/prefix1/59.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>380</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/33.png</normaloff>:/new/prefix1/33.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>291</x>
        <y>380</y>
        <width>90</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/45.png</normaloff>:/new/prefix1/45.png</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>20</y>
       <width>561</width>
       <height>491</height>
      </rect>
     </property>
     <property name="title">
      <string>Livre Table</string>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>521</width>
        <height>401</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_load">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>440</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>Load </string>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/new/prefix1/42.png</normaloff>:/new/prefix1/42.png</iconset>
      </property>
     </widget>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>groupBox</zorder>
    <zorder>label_status_livre_bd</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/new/prefix1/55.png</normaloff>:/new/prefix1/55.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Stats</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_stats_livre">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>480</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{
background-color : #181DA4 ;
border-radius : 20px ;
color : white ;
}</string>
     </property>
     <property name="text">
      <string>SHOW</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/new/prefix1/55.png</normaloff>:/new/prefix1/55.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>1001</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
